lib/commands/app/create.ts

```diff
@@ -75,30 +88,58 @@ export default class AppCreateCmd extends Command {
 			AppCreateCmd,
 		);
 
-		const balena = getBalenaSdk();
-
-		// Create application
+		// Ascertain device type
 		const deviceType =
 			options.type ||
 			(await (await import('../../utils/patterns')).selectDeviceType());
-		let application: BalenaSDK.Application;
+
+		// Ascertain organization
+		const organization =
+			options.organization?.toLowerCase() || (await this.getOrganization());
+
+		// Create application
+		let application: Application;
 		try {
-			application = await balena.models.application.create({
+			application = await getBalenaSdk().models.application.create({
 				name: params.name,
 				deviceType,
-				organization: (await balena.auth.whoami())!,
+				organization,
 			});
 		} catch (err) {
-			// BalenaRequestError: Request error: Unique key constraint violated
 			if ((err.message || '').toLowerCase().includes('unique')) {
+				// BalenaRequestError: Request error: "organization" and "app_name" must be unique.
+				const slug = `${organization}/${params.name}`.toLowerCase();
+				throw new ExpectedError(`Error: application "${slug}" already exists`);
+			} else if ((err.message || '').toLowerCase().includes('unauthorized')) {
+				// BalenaRequestError: Request error: Unauthorized
 				throw new ExpectedError(
-					`Error: application "${params.name}" already exists`,
+					`Error: You are not authorized to add applications to organization "${organization}".`,
```

Comment:
Maybe worth staying consistent in terms? I don't know if what I suggest is proper English though :)

```suggestion
					`Error: You are not authorized to create applications in organization "${organization}".`,
```